cmd/logger.go

```diff
@@ -92,16 +106,21 @@ func (log *Logger) EnableQuiet() {
 	log.quiet = true
 }
 
+// EnableJSON - outputs logs in json format.
+func (log *Logger) EnableJSON() {
+	log.json = true
+}
+
 // Println - wrapper to console.Println() with quiet flag.
 func (log *Logger) Println(args ...interface{}) {
-	if !log.quiet {
+	if !log.quiet && !log.json {
 		console.Println(args...)
 	}
 }
 
 // Printf - wrapper to console.Printf() with quiet flag.
 func (log *Logger) Printf(format string, args ...interface{}) {
-	if !log.quiet {
+	if !log.quiet && !log.json {
```

Comment:
isn't `quiet` overriding `json`?  unable to understand why this check is needed.

Reply:
We do not want to print comments like `Config has been Created` etc to be displayed, irrespective of whether `quiet` flag is set or not, if `json` mode flag is set

Reply:
then do `EnableQuiet()` when `EnableJSON()`?

Reply:
You mean, internally?

Reply:
yes

Reply:
done